src/components/TargetsMap/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, memo } from 'react';
+import { usePosition } from 'use-position';
+import { useSelector, useDispatch } from 'react-redux';
+import { getTargets } from 'redux/actions/targetAction';
+import { getTopics } from 'redux/actions/topicAction';
+
+import './TargetsMap.scss';
+import currentLocation from 'assets/current_location.svg';
+import GoogleMap from './GoogleMap';
+import TargetIndicator from './TargetIndicator';
+import Indicator from './Indicator';
+
+const zoom = 15;
+
+const TargetsMap = () => {
+  const { latitude, longitude } = usePosition(true);
+  const targets = useSelector(({ target: { values } }) => values);
+  const topics = useSelector(({ topic: { values } }) => values);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getTargets(1));
```

Comment:
What's this `1`?

Reply:
According to the api doc, it's the page number but it seems not to be working.

Reply:
Maybe, it's a good idea to create a constant with that. wdyt?

Reply:
Agree. It must have a semantic value.